packages/native/storybook/stories/Loader/InfiniteLoader.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { storiesOf } from "../storiesOf";
+import { InfiniteLoader } from "../../../src";
+import { color, number } from "@storybook/addon-knobs";
+
+const description = `
```

Comment:
The description is broken, I think you need to:

- put blank lines between sections
- for code one liners, add a blank line otherwise syntax highlighting will not work (storybook issue)

<img width="1089" alt="Capture d’écran 2021-12-16 à 09 23 46" src="https://user-images.githubusercontent.com/3428394/146334795-d7ebea7b-c6ee-4bbb-9306-08ef18626fb2.png">

Reply:
I was a little too quick, thanks, corrected